bluepysnap/network.py

```diff
@@ -159,14 +159,7 @@ def get(self, group=None, properties=None):
         if unknown_props:
             raise BluepySnapError(f"Unknown properties required: {unknown_props}")
 
-        # Retrieve the dtypes of the selected properties.
-        # However, the int dtype may not be preserved if some values are NaN.
-        dtypes = {
-            column: dtype
-            for column, dtype in self.property_dtypes.items()
```

Comment:
Should we remove the whole function definition for `property_dtypes` on the `network` level?

Reply:
I think so, I don't find it useful anymore